Fix Display Owner error

Caused by: org.springframework.expression.spel.SpelEvaluationException: EL1008E: Property or field 'type' cannot be found on object of type 'vlad.springframework.vladpetclinic.model.Pet' - maybe not public or not valid?
	at org.springframework.expression.spel.ast.PropertyOrFieldReference.readProperty(PropertyOrFieldReference.java:217)
	at org.springframework.expression.spel.ast.PropertyOrFieldReference.getValueInternal(PropertyOrFieldReference.java:104)
	at org.springframework.expression.spel.ast.PropertyOrFieldReference.access$000(PropertyOrFieldReference.java:51)
	at org.springframework.expression.spel.ast.PropertyOrFieldReference$AccessorLValue.getValue(PropertyOrFieldReference.java:406)
	at org.springframework.expression.spel.ast.CompoundExpression.getValueInternal(CompoundExpression.java:92)
	at org.springframework.expression.spel.ast.SpelNodeImpl.getValue(SpelNodeImpl.java:112)
	at org.springframework.expression.spel.standard.SpelExpression.getValue(SpelExpression.java:338)
	at org.thymeleaf.spring5.expression.SPELVariableExpressionEvaluator.evaluate(SPELVariableExpressionEvaluator.java:265)
	... 93 more

--- a/pet-clinic-data/src/main/java/vlad/springframework/vladpetclinic/model/PetType.java
+++ b/pet-clinic-data/src/main/java/vlad/springframework/vladpetclinic/model/PetType.java
@@ -13,7 +13,12 @@ import javax.persistence.Table;
 @Builder
 @Entity
 @Table(name = "types")
-public class PetType extends BaseEntity{
+public class PetType extends BaseEntity {
     @Column(name = "name")
     private String name;
+
+    @Override
+    public String toString() {
+        return name;
+    }
 }

--- a/pet-clinic-web/src/main/resources/templates/owners/ownerDetails.html
+++ b/pet-clinic-web/src/main/resources/templates/owners/ownerDetails.html
@@ -49,7 +49,7 @@
         <dd
                 th:text="${#temporals.format(pet.birthDate, 'yyyy-MM-dd')}" /></dd>
         <dt>Type</dt>
-        <dd th:text="${pet.type}" /></dd>
+        <dd th:text="${pet.petType}" /></dd>
       </dl>
     </td>
     <td valign="top">

--- a/pet-clinic-web/target/classes/templates/owners/ownerDetails.html
+++ b/pet-clinic-web/target/classes/templates/owners/ownerDetails.html
@@ -49,7 +49,7 @@
         <dd
                 th:text="${#temporals.format(pet.birthDate, 'yyyy-MM-dd')}" /></dd>
         <dt>Type</dt>
-        <dd th:text="${pet.type}" /></dd>
+        <dd th:text="${pet.petType}" /></dd>
       </dl>
     </td>
     <td valign="top">

